frontend/packages/kubevirt-plugin/src/components/modals/vm-description-modal/vm-description-modal.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { TextArea } from '@patternfly/react-core';
+
+import { HandlePromiseProps, withHandlePromise } from '@console/internal/components/utils';
+import {
+  createModalLauncher,
+  ModalTitle,
+  ModalBody,
+  ModalSubmitFooter,
+  ModalComponentProps,
+} from '@console/internal/components/factory';
+import { k8sPatch } from '@console/internal/module/k8s';
+
+import { VMLikeEntityKind } from '../../../types';
+import { getDescription, getVmLikeModel } from '../../../selectors/selectors';
+import { getUpdateDescriptionPatches } from '../../../k8s/patches/vm/vm-patches';
+
+import './_vm-description-modal.scss';
```

Comment:
:+1: on loading relevant styles as part of the React component module. This works well with webpack code splitting feature.